sql/driver/wire.proto

```diff
@@ -78,6 +76,9 @@ message Datum {
 
 // A Result is a collection of rows.
 message Result {
+  // Error is non-nil if an error occurred while executing the statement.
```

Comment:
nit: kinda weird that the nested message `Row` comes after this; why not keep it up top?